articles/quickstart/native/ios-objc/07-linking-accounts.md

```diff
@@ -79,9 +53,9 @@ NSString *otherUserToken = ... // the idToken from the account you want to link
 }];
 ```
 
-## Retrieve Linked Accounts
+## Retrieve the Linked Accounts
 
-Linked accounts, a.k.a. the user's identities, can be retrieved by fetching the user's profile, a process that we already know from the [User Sessions Tutorial](/quickstart/native/ios-objc/03-user-sessions#validate-an-accesstoken):
+You can retrieve the linked accounts (user identities). To achieve this, fetch the user's profile as shown in the [User Sessions](/quickstart/native/ios-objc/03-user-sessions#validate-an-accesstoken) tutorial:
```

Comment:
This is outdated now, the second section covers this https://github.com/auth0/docs/pull/5220/files#diff-c0c5a36c589831186843fa63f3d67751R74 using the MGMT API.

Reply:
Just to make sure, should this paragraph be removed entirely?

Reply:
Yes